Fri, 03 May 2019 14:18:03  --  ./main.py  --  Error on LDAP connection (check IP, DN)
Fri, 03 May 2019 14:20:03  --  ./main.py  --  Error on DATABASE connection (check Host, User, Passwd and Db)
